src/cue_notes_create.lua

```diff
@@ -54,9 +75,10 @@ function show_error(error_code)
         empty_region = "Please select a region\nwith some notes in it!",
         no_notes_in_source_layer = "The music selected contains\nno notes in layer " .. config.source_layer,
         first_make_expression_category = "You must first create a new Text Expression Category called \""..config.cue_category_name.."\" containing at least one entry",
+        no_cue_notes = "The region selected contains \nno cue notes in layer " .. config.cuenote_layer
     }
     local msg = errors[error_code] or "Unknown error condition"
-    finenv.UI():AlertNeutral("script: " .. plugindef(), msg)
+    finenv.UI():AlertInfo("script: " .. plugindef(), msg)
```

Comment:
@cv-on-hub Shouldn't the `msg` param come first?

Reply:
The **second** string parameter (the optional "Title") is bigger and bolder. I wan't the "message" to be bigger and bolder so place it second. In this usage, at least, I'm only adding the name of the script as a courtesy. Not so important.

Reply:
It's up to you, of course, but on Windows the dialog title is neither bigger nor bolder. It also doesn't support line breaks, so you get text like "regionwith". And from a usability perspective, it seems unusual to look at the title bar of a dialog to get the important information.

![](https://user-images.githubusercontent.com/5248041/221297076-7cdba947-229b-468e-bd64-ddcd68277b3c.png)

Reply:
Thanks for letting me know. That's really ugly. Will remove the script name and move the message to the place where Windows can handle it.

Reply:
Thanks, lgtm